src/main/java/frc/robot/commands/led/DefaultLEDCommand.java

```diff
@@ -0,0 +1,90 @@
+package frc.robot.commands.led;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.LEDPattern;
+import edu.wpi.first.wpilibj.RobotState;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.LEDSubsystem;
+
+/**
+ * The default command for controlling the LEDs
+ */
+public class DefaultLEDCommand extends Command {
+
+  private enum LEDMode {
+    DISCONNECTED,
+    DISABLED,
+    DEFAULT
+  }
+
+  private static final LEDPattern PATTERN_OFF = LEDPattern.solid(Color.kBlack);
+
+  private LEDSubsystem ledSubsystem;
+
+  private Timer timer = new Timer();
+  private boolean candyCaneState = false;
+  private static final double CANDY_CANE_SPEED = 1.0;
```

Comment:
The old animations had 1 second for the disabled and test mode, but 0.5 seconds for disconnect, but I'll allow :)